DNN Platform/Library/Common/Initialize.cs

```diff
@@ -455,7 +455,6 @@ private static string InitializeApp(HttpApplication app, ref bool initialized)
                     Globals.DatabaseEngineVersion = GetDatabaseEngineVersion();
 
                     // Try and Upgrade to Current Framewok
```

Comment:
Looks good to me except this little stray comment that is no longer relevant

```suggestion
                    Globals.DatabaseEngineVersion = GetDatabaseEngineVersion();
```